dev_docs/tutorials/building_a_kibana_distributable.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: kibDevTutorialBuildingDistributable
+slug: /kibana-dev-docs/tutorial/building-distributable
+title: Building a Kibana distributable
+summary: Learn how to build a Kibana distributable
+date: 2021-05-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'build', 'distributable']
+---
+
+On the course of that tutorial it will be possible to learn more about how we can create a Kibana distributable 
+as well as handle its different main configurations.
+
+At any given point, you can get CLI help running the following command:
+
+```bash
+yarn build --help
+```
+
+## Prerequisites
+
+For the general aspects of the build we only require you to use one of the following operating systems:
+
+- Linux
+- macOS
+
+In case you want to build OS Packages there are other things to have in mind. Packages are built using fpm, dpkg, rpm, and Docker.
+Please make sure you have those dependencies installed using:
+
+```bash
+apt-get install ruby ruby-dev rpm dpkg build-essential
+gem install fpm -v 1.5.0
+```
+
+For Docker, the installation instructions can be found at [Install Docker Engine](https://docs.docker.com/engine/install/).
+
+> NOTE: OS Packages building has only been tested on Linux and may not be fully supported on any other platform. 
+
+
+## Create a simple Kibana distributable
+
+In a great majority of the use cases where we need to build a Kibana distributable, we can live with a simple version of it.
```

Comment:
thoughts on reversing the terminology here?  It's not 100% clear to me what constitutes OS packages on first pass, noting this builds zip and tar archives only may be more clear.